pkg/apis/networking/register.go

```diff
@@ -33,4 +33,12 @@ const (
 	// value a different reconcilation logic may be used (for examples,
 	// Istio-based ClusterIngress will reconcile into a VirtualService).
 	IngressClassAnnotationKey = "networking.knative.dev/ingress.class"
+
+	// RouteLabelKey is the label key attached to ClusterIngress resources
+	// to indicate which Route triggered their creation.
+	RouteLabelKey = GroupName + "/route"
```

Comment:
This feels like a layering violation.  I think I'd expect:
```
serving.knative.dev/route: foo
```

I would like to avoid the Ingress resources knowing anything about Routes or our other resources.

Reply:
Aha, I ignored the fact that ingress and route resource are not in a same group.
So move this definition to the register file of route?

Reply:
Yeah that SGTM